populate-scripts/update-db.sql

```diff
@@ -10622,45 +10629,33 @@ UPDATE tasks set
                         "fr": "Questionnaire"
                     },
                     "questions":[
-                        {
```

Comment:
moved the description of each word (gambling, sex, buying, etc... ) to below the word itself